modules/lib/lib-scheduler/src/main/java/com/enonic/xp/lib/scheduler/GetScheduledJobLastRunStateHandler.java

```diff
@@ -0,0 +1,33 @@
+package com.enonic.xp.lib.scheduler;
+
+import java.util.Optional;
+
+import com.google.common.base.Preconditions;
+
+import com.enonic.xp.lib.scheduler.mapper.ScheduledJobLastRunStateMapper;
+import com.enonic.xp.scheduler.ScheduledJobName;
+
+public final class GetScheduledJobLastRunStateHandler
```

Comment:
After a second thought I guess this makes js and java API asymmetrical.
either split methods in Java API or join methods in js